daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto

```diff
@@ -308,6 +309,22 @@ enum PrimType {
   // Builtin type 'TGenMap`
   // *Available in versions >= 1.dev*
   GENMAP = 20;
+
+  // Builtin type 'TAnyException'
+  // *Available in versions >= 1.dev*
+  ANY_EXCEPTION = 21;
+
+  // Builtin type 'TGeneralError'
+  // *Available in versions >= 1.dev*
+  GENERAL_ERROR = 22;
+
+  // Builtin type 'TArithmeticError'
+  // *Available in versions >= 1.dev*
+  ARITHMETIC_ERROR = 23;
+
+  // Builtin type 'TContractError'
+  // *Available in versions >= 1.dev*
+  CONTRACT_ERROR = 24;
```

Comment:
I am a bit worried the number of exception types will explode in the future.  I wondering if it would not be a good idea to defined a new primitive type specifically for them.